android/titanium/src/java/org/appcelerator/titanium/view/TiDrawableReference.java

```diff
@@ -973,4 +1203,171 @@ public String getUrl()
 	{
 		return url;
 	}
+
+
+	/**
+	 * Private class used to fetch GPU information such as the max texture size.
+	 * <p>
+	 * You cannot create instances of this class. Instead, you are expected to call its static functions.
+	 */
+	private static class GpuInfo
```

Comment:
This class should probably be moved into it's own file:
`android/titanium/src/java/org/appcelerator/titanium/util/TiGpuInfo.java`